docs/src/main/sphinx/object-storage/file-formats.md

```diff
@@ -108,7 +100,7 @@ with Parquet files performed by supported object storage connectors:
     `parquet_small_file_threshold`.
   - `3MB`
 * - `parquet.experimental.vectorized-decoding.enabled`
-  - Enable using Java Vector API for faster decoding of parquet files. 
+  - Enable using Java Vector API for faster decoding of parquet files.
```

Comment:
```suggestion
  - Enable using Java Vector API (SIMD) for faster decoding of parquet files.
```

Reply:
oh .. thats an unrelated formatting only update..

Reply:
but we can probably add anyway